### PR TITLE
Add date range support to --transactions

### PR DESCRIPTION
**Pull request by burkematthew**

### CODE REVIEW

File: mintapi/api.py [hunk 1]
```diff
@@ -860,7 +860,7 @@ def get_detailed_transactions(self, include_investment=False,
 
         return df
 
-    def get_transactions_csv(self, include_investment=False, acct=0):
+    def get_transactions_csv(self, include_investment=False, start_date=None, end_date=None, acct=0):
```
[the remark this answers is not in the document]
Comment (burkematthew) [reply mintapi/api.py:873]: @mrooney  @jaron-l while I see the benefit of trying to get the integration test implemented in conjunction with this enhancement, I think we have some details to work out on how we want to further implement those style of tests.  So I'd like to propose that if there are no issues with this PR, we merge it and map out integration test details\process on #311 .

[the remark this answers is not in the document]
Comment (mrooney) [reply mintapi/api.py:873]: @burkematthew I'm good with that, given the lack of infrastructure currently for it!

**Comment by mrooney** [comment]
Looks nice! Since we didn't end up testing, I just wanted to verify that you've manually tested w/ and w/o these arguments to make sure the original case still works. Also, does it work if you only specify one, or do you need both if you specify one? Thanks again for this contribution!

**Comment by burkematthew** [comment]
@mrooney I manually verified the following:

* A date range is not required.
* Only using the starting date returns the correct range.
* Only using the ending date returns the correct range.
* Using both the starting date and the ending date returns the correct range.

**Review by mrooney — APPROVED** [review]
LGTM, thanks!

**Pull request merged**